index.html

```diff
@@ -74,49 +69,47 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
                                   <div class="float-left">
                                       <img src="/images/icon-download-g.svg" class="os-icon os-icon-g">
                                       <img src="/images/icon-download-w.svg" class="os-icon os-icon-w">
-                                      {{ item.buttonDownloads }}
+                                      {% t homepage_content.buttonDownloads %}
                                   </div>
                                   <div class="float-right">
-                                      <a href="/{{ page.lang }}/downloads">{{ item.buttonGetBisq }}</a>
+                                      <a href="/downloads">{% t homepage_content.buttonGetBisq %}</a>
                                   </div>
                               </div>
                           </button>
 
                           <button type="button" class="btn btn-lg btn-secondary dropdown-toggle dropdown-toggle-split hero-btn shadow-btn " data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                              <span class="sr-only">{{ item.buttonToggle }}</span>
+                              <span class="sr-only">{% t homepage_content.buttonToggle %}</span>
                           </button>
 
                           <div class="dropdown-menu dropdown-menu-right shadow-lg">
-                              {% include os_selector_options_tr.html %}
+                              {% include os_selector_options.html %}
                           </div>
-<!-- Need to repeat the function because the if/else function seems to nullify it -->
-{% assign item = site.data.en.homepage_content %}
                       </div>
 
 
-                      <a href="/{{ page.lang }}/downloads" class="downloads-mobile btn btn-lg btn-outline-secondary btn-icon hero-btn col-sm-12 col-md-12 col-lg-3 px-0 shadow-btn ">
+                      <a href="/downloads" class="downloads-mobile btn btn-lg btn-outline-secondary btn-icon hero-btn col-sm-12 col-md-12 col-lg-3 px-0 shadow-btn "> <!-- TODO -->
```

Comment:
Looks like the front-page download button always goes to the English version of the Downloads page when another language is set.

Reply:
Forgotten TODO :) Fixing it and taking advantage of this to fix also the link of the button.